### DISCUSSION TITLE
menu system with more then four menus?

### DISCUSSION THREAD

**Discussion by TowerDefender007** [VasilKalchev/LiquidMenu#101] [Q&A]
I came to the problem that I want to implement more then 4 menus into a menu system. My general idea was to have the top menu that consists aout of two lines after you get into one of those you get another two or more lines to decide about. After that you get into the general menu of those functions. Now I made it different so I dont have to use the top menu but now I would need another menu in my menu system and I cant again. SO my question would be is it possible to have more then four menus in a menu system?

**Comment by TowerDefender007** [comment]
And also maybe implement the was to only clear a line not the complete screen with the linename.update();.

**Comment by VasilKalchev** [comment]
Hello @TowerDefender007.

The `LiquidSystem` constructor is limited for up to 4 menus, but you can add more menus by calling `bool LiquidSystem::add_menu(LiquidMenu &liquidMenu)` after.

By default there is a limitation for 8 menus in a menu system. But this can be changed in: https://github.com/VasilKalchev/LiquidMenu/blob/d58255e1fd4fbeefad5b6fdf70c80c6460696021/src/LiquidMenu_config.h#L73
